src/autosuggest/internal.tsx

```diff
@@ -179,7 +180,7 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
   const highlightedOptionIdSource = useUniqueId();
   const highlightedOptionId = autosuggestItemsState.highlightedOption ? highlightedOptionIdSource : undefined;
 
-  const isEmpty = !value && !autosuggestItemsState.items.length;
+  const isEmpty = !autosuggestItemsState.items.length;
```

Comment:
should this change be made only for the case when hideEnteredTextLabel=true?

Reply:
Based on my testing and understanding of the [options controller](https://github.com/cloudscape-design/components/blob/main/src/autosuggest/options-controller.ts#L70), I've found that when `value` is defined and `hideEnteredTextLabel` is false, `autosuggestItemsState.items` always includes at least one item—the entered text label. Therefore, I've removed the `!value`. Please let me know if this works.